src/box/index_def.h

```diff
@@ -155,9 +165,28 @@ index_opts_is_equal(const struct index_opts *o1, const struct index_opts *o2)
 		return false;
 	if (o1->hint != o2->hint)
 		return false;
+	if (o1->covered_count != o2->covered_count)
+		return false;
+	for (uint32_t i = 0; i < o1->covered_count; i++) {
+		if (o1->covered[i] != o2->covered[i])
+			return false;
+	}
 	return 0;
 }
 
+/**
+ * Returns whether `fieldno` is one of the covered fields.
+ */
+static inline bool
+index_opts_is_covered(const struct index_opts *opts, uint32_t fieldno)
```

Comment:
The function isn't used anywhere.

Reply:
Originally it is used in EE. Now it is used in `luaT_index_push_covers()` also.